src/Umbraco.Core/Services/IRelationService.cs

```diff
@@ -341,5 +341,15 @@ public interface IRelationService : IService
         /// </summary>
         /// <param name="relationType"><see cref="IRelationType"/> to Delete Relations for</param>
         void DeleteRelationsOfType(IRelationType relationType);
+
+        /// <summary>
+        /// Gets a relation by the unique combination of parentId, childId and relationType.
+        /// </summary>
+        /// <param name="parentId">The id of the parent item.</param>
+        /// <param name="childId">The id of the child item.</param>
+        /// <param name="relationType">The RelationType.</param>
+        /// <returns>The relation or null</returns>
+        IRelation GetByParentChildAndType(int parentId, int childId, IRelationType relationType);
```

Comment:
Should be named GetByParentAndChild to be consistent with the `GetByParentOrChildId` method